packages/app/src/providers/hsds/hsds-api.ts

```diff
@@ -108,17 +108,15 @@ export class HsdsApi extends ProviderApi {
   }
 
   public async getValue(params: ValuesStoreParams): Promise<unknown> {
-    const { path } = params;
+    const { dataset } = params;
+    assertHsdsDataset(dataset);
```

Comment:
We still have to make sure here that the dataset includes the HSDS-specific `id` property. It's for TS, but also to ensure that the `id` property of the dataset doesn't get lost from the time it is returned by `getEntity` to the time it arrives here.

Reply:
The alternative to this implicit coupling between `getEntity` and `getValue` would be to keep a private map between entity paths and IDs. I don't think it's worth the complexity.

Reply:
> I don't think it's worth the complexity.

Yeah agreed. I can live with this.